bootstrap/standalone/src/main/java/org/geysermc/platform/standalone/GeyserStandaloneBootstrap.java

```diff
@@ -106,7 +139,7 @@ public void onEnable() {
         LoopbackUtil.checkLoopback(geyserLogger);
         
         try {
-            File configFile = FileUtils.fileOrCopiedFromResource("config.yml", (x) -> x.replaceAll("generateduuid", UUID.randomUUID().toString()));
```

Comment:
This causes an issue when you specify a config file name that doesn't exist yet as it will attempt to pull that name from the jar resources but it wont exist. Likely needs another method with an extra argument specifying the file within the jar.

Reply:
Maybe we can change the method to find a hardcoded "config.yml" from the jar if the file doesn't exist?

Reply:
Yeah, I see what you mean. I'll take a look.

Reply:
I think this is fairly easily fixed, there's already an overload that takes a file object and name separately.